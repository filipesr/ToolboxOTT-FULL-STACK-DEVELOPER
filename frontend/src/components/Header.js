function Header({ title = "React Test App" }) {
  return (
    <h4 style={{ color: "white", backgroundColor: "#f66", padding: "1rem" }}>
      {title}
    </h4>
  );
}

export default Header;
