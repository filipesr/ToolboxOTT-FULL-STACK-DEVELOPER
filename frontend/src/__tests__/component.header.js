import React from "react";
import Header from "../components/Header";
import { create } from "react-test-renderer";

describe("Header coponent", () => {
  test("testing list files", () => {
    let tree = create(<Header />);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
