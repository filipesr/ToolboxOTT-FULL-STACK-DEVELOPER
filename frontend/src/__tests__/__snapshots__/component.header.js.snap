// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header coponent testing list files 1`] = `
<h4
  style={
    Object {
      "backgroundColor": "#f66",
      "color": "white",
      "padding": "1rem",
    }
  }
>
  React Test App
</h4>
`;
